### DISCUSSION TITLE
How to render katext in textarea, input ?

### DISCUSSION THREAD

**Discussion by sontt-1215** [KaTeX/KaTeX#2705] [Q&A]
Hi everybody !!!
I have a problem ! I want to render katex in textarea but i can't do it. (I use vue)
This is my textarea
```
<textarea class="w-100" v-katex:auto="options"> {{ form.selectedKatex }}</textarea>
```
This is my options
```
data() {
    return {
        options: {
          delimiters: [
            { left: "$$", right: "$$", display: true },
            { left: "$", right: "$", display: true },
            { left: "\\(", right: "\\)", display: true },
            { left: "\\[", right: "\\]", display: true },
          ],
          ignoredTags: ["script", "noscript", "style", "pre", "code"],
      },
   }
}
```
Please help me !!!

**Comment by edemaine** [comment]
Are you using https://github.com/lucpotage/vue-katex? If so, I think the question would be better asked there, as I'm unfamiliar with how it works.

**Comment by sontt-1215** [comment]
@edemaine thanks for reply :(
But there is also no idea, i think I'll have to do it myself

**Comment by k4b7** [comment]
I don't think this is possible since `textarea` only support plaintext for its content.  You might be able to use a contenteditable div somehow.
